app/models/project.rb

```diff
@@ -1,4 +1,18 @@
 class Project < ApplicationRecord
   validates_presence_of :name, :material
   belongs_to :challenge
+  has_many :contestant_projects
+  has_many :contestants, through: :contestant_projects
+
+  def challenge_theme
+    challenge.theme
+  end
+
+  def num_contestants
+    contestants.count
```

Comment:
Excellent work encapsulating all this logic into model methods! Even though these are simple calculations, they're still calculations so by strict MVC conventions, they should live at the model level